### ISSUE TITLE
CSV input with timestamp schema support

### ISSUE THREAD

**Issue by tooptoop4** [YotpoLtd/metorikku#159]
I am trying to load in a csv input and have put following json schema but i can't see anyway for metorikku to interpret the timestamp format (ie yyyy-mm-dd hh:   ..etc) of the csv column in order to convert to spark timestamp 

{
  "$schema": "smallTestSchema",
  "id": "smallTestSchema",
  "type": "object",
  "name": "/",
  "properties": {
    "row_no": {      "id": "smallTestSchema/row_no/",      "type": "string",      "name": "row_no"    }
    ,"start_date": {      "id": "smallTestSchema/start_date/",      "type": "string",      "name": "start_date"    }
    ,"end_date": {      "id": "smallTestSchema/end_date/",      "type": "string",      "name": "end_date"    }
    ,"unit_name": {      "id": "smallTestSchema/unit_name/",      "type": "timestamp",      "name": "unit_name"    }
}

**Comment by lyogev** [comment]
You are right dates are not supported currently in our Json schema implementation.
As a workaround load it as string and use SQL function to_timestamp or to_date as documented here:
https://spark.apache.org/docs/2.4.0/api/sql/
